_benchmark/index.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+title: OpenSearch Benchmark
+nav_order: 1
+has_children: false
+has_toc: false
+---
+
+# OpenSearch Benchmark
+
+OpenSearch Benchmark is a macrobenchmark utility provided by the [OpenSearch Project](https://github.com/opensearch-project). You can use OpenSearch Benchmark to gather performance metrics from an OpenSearch cluster for a variety of purposes, including:
+
+- Tracking the overall performance of an OpenSearch cluster.
+- Informing decisions about when to upgrade your cluster to a new version.
+- Determining how changes to your workflow&#8212;such as modifying mappings or queries&#8212;might impact your OpenSearch cluster.
+
+OpenSearch Benchmark can be installed directly on a compatible host running Linux and macOS. You can also run OpenSearch Benchmark in a Docker container. See [Installing OpenSearch Benchmark]({{site.url}}{{site.baseurl}}/benchmark/installing-benchmark/) for more information.
+
+## OpenSearch Benchmark documentation status
```

Comment:
Is this section necessary? It essentially states the obvious, and I would prefer that we keep this kind of content out of the documentation.

Reply:
I'm with you. I'll remove it after I address the other comments.